CHANGES.rst

```diff
@@ -3,9 +3,108 @@
 0.14 (unreleased)
 -----------------
 
-- Planned for September 15, 2019
+Summary
++++++++
+
+- Released on September 30th, 2019
+- 8 contributors
+- 92 pull requests (not all listed below)
+
+**What's new**
+
+Gammapy v0.14 features a new high level analysis interface. Starting from
+a YAML configuration file, it supports the standard use-cases of joined
+or stacked 3D as well as 1D reflected region analyses. It also supports
+computation of flux points for all cases. The usage of this new ``Analysis``
+class is demonstrated in the ``hess.ipynb`` tutorial.
+
+To bundle all model classes and the functionality from ``gammapy.utils.fitting``
```

Comment:
Since this will require some effort from users to change their scripts, I would add a sentence at the end: 
e.g. While these changes will require users to adapt their scripts to the new import structure and naming scheme, but this provides a much clearer interface.

Reply:
You could put all structure changes together. First the removal of `background` and `image`, then the introduction of `modeling`

Reply:
Done